.gitignore

```diff
@@ -2,3 +2,4 @@ vendor
 composer.phar
 composer.lock
 phpunit.xml
+.idea
```

Comment:
Could you please remove this line? You should add this line in your Git global ignore.